src/app/models/obstacle.ts

```diff
@@ -0,0 +1,9 @@
+export class Obstacle {
```

Comment:
why do you need this for the navbar?

Reply:
this actually doesnt used in the mocked dummy page but was made in early with all the models in detail.